tests/README.md

```diff
@@ -83,7 +83,19 @@ The testing machines can run on Openshift cluster(s).
 
 Create a service account for use by the testing machines. Make sure to have the
 ```oci-kvm-hook``` package installed on all nodes.  This is because of the requirement
-to access ```/dev/kvm```. Further work is necessary to remove this requirement.
+to access ```/dev/kvm```.
+
+This creates all the remaining kubernetes objects. The secrets are created from the
+```/var/lib/cockpit-tests/secrets``` directory as described above.
+
+    $ sudo make tests-secrets | oc create -f -
+    $ oc create -f tests/cockpit-tasks-restricted.json
+
+## High Density Openshift Deployment
```

Comment:
This part would apply to non-restricted environments like our current cockpit openshift cluster? It's not entirely clear whether this is mutually exclusive with the above -restricted.json.

Reply:
I've run them both on the same cluster successfully. One simply uses less efficient mechanisms when we don't have the right SCC to efficiently cache images across pods.

Reply:
You can deploy the same stuff in the same cluster. Obviously the names conflict, but you can put it in another project.